wp/wp-content/themes/sampletheme/functions/acf.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+class AcfCustomJsonData
+{
+  private static $acf_custom_json_data;
+
+  private static $theme_path = '/inc/acf-json';
+
+  private function __construct()
+  {
+  }
+
+  public static function get_json_data_path($default_path)
```

Comment:
This should not have such a global access modifier.

Please make it _private_ along with moving `my_acf_json_save_point` and other functions into this class.